app/design/frontend/base/default/template/TIG/MyParcel2014/checkout/onepage/shipping_method/available.phtml

```diff
@@ -192,7 +192,7 @@ $toggleOptions = $helper->getConfig('toggle_options', 'checkout') == '1' ? true
                                         </div>
                                     </td>
                                 </tr>
-                                <tr>
+                                <tr id="method-myparcel-delivery-standard-div">
```

Comment:
dit is geen div. beetje gek om de naam ernaar te noemen

Reply:
Done!